products/BellHybrid/BellHybridMain.cpp

```diff
@@ -50,8 +50,7 @@ int main()
 {
     constexpr auto ApplicationName = "BellHybrid";
 
-    const std::vector<std::string> fileIndexerAudioPaths = {
-        {purefs::dir::getCurrentOSPath() / "assets/audio/bg_sounds"}};
+    std::vector<std::string> fileIndexerAudioPaths = {{purefs::dir::getCurrentOSPath() / "assets/audio/bg_sounds"}};
```

Comment:
I don't get why it shouldn't be const

Reply:
It's the same issue as in ColorTestWindow.cpp - this variable is later passed to std::move:

`systemServices.emplace_back(sys::CreatorFor<service::ServiceFileIndexer>(std::move(fileIndexerAudioPaths)));`

If it's const, the std::move will have no effect.

Reply:
cant we in this case just remove std::move there?

Reply:
Yeah, that seems to be a better solution, as the performance boost from using std::move in this place is rather insignificant. Changed according to your suggestion.